app/routes/application.js

```diff
@@ -1,8 +1,7 @@
 import { inject as service } from '@ember/service';
 import { set, get, action } from '@ember/object';
 import Route from '@ember/routing/route';
-import Ember from 'ember';
-
+const Ember = require('ember')['default'];
```

Comment:
I just wonder why this move ? Isn't the recommended way to `import Ember from 'ember` ?

Reply:
@pzuraq I thought `import Ember from 'ember'` counted as global?